tgui-next/packages/tgui/interfaces/Cryo.js

```diff
@@ -0,0 +1,123 @@
+import { Fragment } from 'inferno';
+import { act } from '../byond';
+import { Button, LabeledList, ProgressBar, Section} from '../components';
+
+export const Cryo = props => {
+  const { state } = props;
+  const { config, data } = state;
+  const { ref } = config;
+  const damageTypes = [
+    {
+      label: "Brute",
+      type: "bruteloss",
+    },
+    {
+      label: "Respiratory",
+      type: "oxyLoss",
+    },
+    {
+      label: "Toxin",
+      type: "toxLoss",
+    },
+    {
+      label: "Burn",
+      type: "fireloss",
+    },
+  ];
+
+  return (
+    <Fragment>
+      <Section
+        title="Occupant">
+        <LabeledList>
+          <LabeledList.Item
+            label="Occupant"
+            content={data.occupant.name ? data.occupant.name : "No Occupant"} />
+          {!!data.hasOccupant && (
+            <Fragment>
+              <LabeledList.Item
+                label="State"
+                content={data.occupant.stat} />
+              <LabeledList.Item
+                label="Temperature"
+                content={data.occupant.bodyTemperature} />
+              <LabeledList.Item
+                label="Health"
+                content={(
+                  <ProgressBar
+                    value={data.occupant.health / (data.occupant.maxHealth - data.occupant.minHealth)}
+                    content="" />)} />
+              <LabeledList.Item
+                content={(damageTypes.map(damageType => (
+                  <LabeledList.Item
+                    key={damageType.id}
+                    label={damageType.label}
+                    content={(
+                      <ProgressBar
+                        value={data.occupant[damageType.type]/100}
+                        content={data.occupant[damageType.type]} />
+                    )} />
+                )))} />
+            </Fragment>
+          )}
+        </LabeledList>
+      </Section>
+      <Section
+        title="Cell">
+        <LabeledList>
+          <LabeledList.Item
+            label="Power"
+            content={(
+              <Button
+                icon={data.isOperating ? "power-off" : "times"}
+                disabled={data.isOpen}
+                onClick={() => act(ref, 'power')}
+                content={data.isOperating ? "On" : "Off"} />
+            )} />
+          <LabeledList.Item
+            label="Temperature"
+            content={data.cellTemperature} K />
```

Comment:
The K here is dangling. Could be solved with something like `content={data.cellTemperature + ' K'}` or look at how Canister.js does it for canister pressure.